download-vs-code.sh

```diff
@@ -228,12 +237,15 @@ if [ "${BIN_TYPE}" = "server" -a ${IS_ALPINE} -eq 1 ]; then
     ARCH="alpine" # Alpine is NOT an Arch but a flavor of Linux, oh well.
 fi
 
-# We hard-code this because all but a few options returns a 404.
-commit_sha=$(get_latest_release "win32" "x64" "${BUILD}")
-
-if [ -z "${commit_sha}" ]; then
-    echo "could not get the VS Code latest commit sha, exiting"
-    exit 1
+if [ -n "${USE_COMMIT}" ]; then
+    commit_sha="${USE_COMMIT}"
+else
+    # We hard-code this because all but a few options returns a 404.
+    commit_sha=$(get_latest_release "win32" "x64" "${BUILD}")
+    if [ -z "${commit_sha}" ]; then
+        echo "could not get the VS Code latest commit sha, exiting"
+        exit 1
```

Comment:
Can you move this if block outside the else and below. That way it will throw an error if you try to enter an empty commit as well.